packages/jest-serializer/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "jest-serializer",
+  "version": "22.3.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/facebook/jest.git"
+  },
+  "license": "MIT",
+  "main": "build/index.js",
+  "dependencies": {
+    "merge-stream": "^1.0.1"
```

Comment:
Where is used this dependency?

Reply:
Nowhere 😳I just copy-pasted the `jest-worker` one and forgot to remove.